src/EA8AE0.c

```diff
@@ -24,7 +24,37 @@ static char* N(exit_str_2) = "kzn_20";
 
 INCLUDE_ASM(s32, "EA8AE0", func_80240B00_EA95E0);
 
-INCLUDE_ASM(s32, "EA8AE0", func_80240BD4_EA96B4);
+typedef struct unk {
+    f32 unk0;
+    f32 unk4;
+    f32 unk8;
+} unk;
+
+ApiStatus func_80240BD4_EA96B4(Evt *script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 temp_s0_5;
+    s32 temp_s0;
+    s32 temp_s1;
+    s32 temp_s2;
+    s32 temp_s3;
+    s32 temp_s4;
+    unk* temp_a0;
+    s32 temp;
+    
+    temp_s3 = dead_evt_get_variable(script, *args++);
```

Comment:
These can be initialized on the same line they're declared. So 
 like `s32 temp_s2 = dead...`